app/scienceapi/events/models.py

```diff
@@ -70,15 +72,26 @@ class Event(models.Model):
         help_text='Slug appended to the Mozilla Science event\'s '
                   'url that represents this event',
     )
+    is_virtual = models.BooleanField(
+        default=False,
+        help_text='Check this box if your event is happening virtually, '
+                  'that is it\'s not tied to a particular location.'
+    )
+
+    ALL_TIMEZONES = [(i, i) for i in pytz.all_timezones]
+    timezone = models.CharField(
+        max_length=40,
+        choices=ALL_TIMEZONES,
+        help_text='Timezone of the place where the event will be held. '
+                  'If the event is virtual select any timezone you like. '
```

Comment:
Change this to "If the event is virtual, use whatever time zone you like and enter the times in that zone."